library/core/src/primitive_docs.rs

```diff
@@ -1104,11 +1104,11 @@ mod prim_usize {}
 /// * [`Clone`] \(Note that this will not defer to `T`'s `Clone` implementation if it exists!)
 /// * [`Deref`]
 /// * [`Borrow`]
-/// * [`Pointer`]
+/// * [`fmt::Pointer`]
 ///
 /// [`Deref`]: ops::Deref
 /// [`Borrow`]: borrow::Borrow
-/// [`Pointer`]: fmt::Pointer
+/// [`fmt::Pointer`]: fmt::Pointer
```

Comment:
This link is unnecessary, it should be auto linked since the text is the same.

Reply:
I've removed this (and similar) links